concreate/cli.py

```diff
@@ -53,6 +53,11 @@ def parse(self):
                                 action='store_true',
                                 help='Run @wip tests only')
 
+        test_group.add_argument('--test-steps-url',
+                                default='https://github.com/jboss-container-images/'
+                                'concreate-test-steps.git',
```

Comment:
This line breaking is more than weird.

Reply:
its how you handle long string in python - multiple strings are concatenated together